conanfile.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os.path
+from conans import ConanFile, CMake
+from conans.tools import Git
+from conans.errors import ConanInvalidConfiguration
+
+class ImguiSFML(ConanFile):
+    """ Imgui-SFML library conan file.
+
+    Options:
+
+    * shared: True or False
+      Whether to build this library shared or static
+
+    * fPIC: True or False
+      Whether or not to use -fPIC option while building
+
+    * imconfig: None or String
+      Use 'None' if you want this library to provide default
+      ImGui's imconfig.h file. Otherwise, provide a path to the
+      imconfig.h file you want to include in the build.
+
+    * imconfig_install_folder: None or String
+      Use 'None' if you want this build to include imconfig.h file,
+      otherwise use anything else.
+
+    * imgui_revision: None or String
+      Tag or branch of ImGui repository that should be patched with
+      ImGui-SFML. 'None' will results in master branch.
+    """
+
+    name = 'ImGui-SFML'
+    version = '2.0.1'
+    description = 'ImGui binding for use with SFML'
+    topics = ('conan', 'sfml', 'gui', 'imgui')
+    url = 'https://github.com/eliasdaler/imgui-sfml'
+    homepage = 'https://github.com/eliasdaler/imgui-sfml'
+    author = 'Elias Daler <eliasdaler@yandex.ru>'
+    build_requires = 'sfml/2.5.1@bincrafters/stable'
+    license = 'MIT'
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    options = {
+        'shared': [True, False],
+        'fPIC': [True, False],
+        'imconfig': 'ANY',
+        'imconfig_install_folder': 'ANY',
+        'imgui_revision': 'ANY'
+    }
+    default_options = {
+        'shared': False,
+        'fPIC': True,
+        'imconfig': None,
+        'imconfig_install_folder': None,
+        'imgui_revision': 'v1.70'
+    }
+    exports_sources = [
+        'imgui-SFML.cpp',
+        'imgui-SFML.h',
+        'imconfig-SFML.h',
+        'imgui-SFML_export.h',
+        'CMakeLists.txt',
+        'cmake/FindImGui.cmake'
+    ]
+    exports = 'LICENSE.md'
+    _imgui_dir = 'imgui'
+
+    def source(self):
+        imgui_revision = str(self.options.imgui_revision) if self.options.imgui_revision else 'master'
```

Comment:
Let's make it an error to set `imgui_revision` to `None`.
It'll force users to set it to 'master' if they want, which will be more predictive.